dao/seeds/source_types.yml

```diff
@@ -302,13 +302,28 @@ oracle-cloud-infrastructure:
             initialValue: ocid
             name: authentication.authtype
           - component: text-field
-            isRequired: true
-            label: Oracle Cloud ID (OCID)
+            hideField: true
+            initializeOnMount: true
+            initialValue: ocid.uuid
             name: authentication.username
+          - component: text-field
+            name: application.extra.bucket
+            label: Bucket Namespace
```

Comment:
Should this be `Bucket Name`

Reply:
Yes I fixed all. I probably got lost in yaml syntax 🤦